editor/gui/editor_dir_dialog.cpp

```diff
@@ -59,13 +55,35 @@ void EditorDirDialog::_update_dir(TreeItem *p_item, EditorFileSystemDirectory *p
 		p_item->set_text(0, p_dir->get_name());
 	}
 
+	if (path == new_dir_path) {
+		p_item->select(0);
+	} else if (!p_item->get_parent()) {
+		p_item->select(0);
+	}
+
 	updating = false;
 	for (int i = 0; i < p_dir->get_subdir_count(); i++) {
 		TreeItem *ti = tree->create_item(p_item);
 		_update_dir(ti, p_dir->get_subdir(i));
 	}
 }
 
+void EditorDirDialog::config(const Vector<String> &p_paths) {
+	ERR_FAIL_COND(p_paths.is_empty());
+
+	if (p_paths.size() == 1) {
+		String path = p_paths[0];
+		if (path.ends_with("/")) {
+			path = path.substr(0, path.length() - 1);
+		}
+		// TRANSLATORS: %s is the file name that will be moved or duplicated.
+		set_title(vformat(TTR("Move/Duplicate: %s"), path.get_file()));
+	} else {
+		// TRANSLATORS: %d is the number of files that will be moved or duplicated.
+		set_title(vformat(TTRN("Move/Duplicate %d Item", "Move/Duplicate %d Items", p_paths.size()), p_paths.size()));
```

Comment:
The single item string will be unused 🤔

Reply:
It's necessary in order to use `TTRN`. Using `TTRN` here for languages that have more complex plural forms.

Reply:
Yep, I get why TTRN is here. I wonder if there is a way to avoid useless string.

Reply:
Actually nevermind 🤦‍♂️